internal/app/config/config.go

```diff
@@ -105,10 +105,12 @@ func pflagNormalizer(_ *pflag.FlagSet, name string) pflag.NormalizedName {
 }
 
 func buildConfig() *Config {
+	quoteReplacer := strings.NewReplacer("\"", "", "'", "")
+
 	return &Config{
 		AllowedOrigins: viper.GetStringSlice(allowedOrigins),
 		Database: Database{
-			ConnectionString: strings.TrimSpace(viper.GetString(databaseConnectionString)),
+			ConnectionString: strings.TrimSpace(quoteReplacer.Replace((viper.GetString(databaseConnectionString)))),
```

Comment:
I guess this should be handled differently. This way, there would be problems if the connection string contains `"` or `'`.

Reply:
Does that even make sense? I guess `"` and `'` could be in a database password, at least.
I'm open to suggestions, maybe it's just an issue, with the deployment setup?

Reply:
I guess this happens because env files are defined pretty weakly.

As it seems, e. g. kustomize (?) would parse lines differently than a bash shell by not interpreting `"`/`'`. I would not try to determine if any of these approaches is the correct one and adapt to it, but just use a fitting syntax for each tooling, respectively.